.github/workflows/build-and-test.yml

```diff
@@ -55,28 +38,10 @@ jobs:
       resultBundle: TestAppiPadOS.xcresult
       destination: 'platform=iOS Simulator,name=iPad mini (6th generation)'
       artifactname: TestAppiPadOS.xcresult
-  watchos:
-    name: Build and Test watchOS
-    uses: StanfordBDHG/.github/.github/workflows/xcodebuild-or-fastlane.yml@v2
-    with:
-      path: 'Tests/UITests'
-      scheme: TestAppWatchApp
-      resultBundle: TestAppWatchApp.xcresult
-      destination: 'platform=watchOS Simulator,name=Apple Watch Series 8 (45mm)'
-      artifactname: TestAppWatchApp.xcresult
-  codeql:
-    name: CodeQL
-    uses: StanfordBDHG/.github/.github/workflows/xcodebuild-or-fastlane.yml@v2
-    with:
-      codeql: true
-      xcodeversion: '14.3.1'
-      scheme: TemplatePackage
-    permissions:
-      security-events: write
-      actions: read
+
   uploadcoveragereport:
     name: Upload Coverage Report
-    needs: [packageios, packagewatchos, packagevisionos, ios, ipados, watchos]
+    needs: [packageios, ios]
     uses: StanfordBDHG/.github/.github/workflows/create-and-upload-coverage-report.yml@v2
     with:
-      coveragereports: TemplatePackage.xcresult TemplatePackageWatchOS.xcresult TemplatePackageVisionOS.xcresult TestApp.xcresult TestAppiPadOS.xcresult TestAppWatchApp.xcresult
+      coveragereports: SpeziLicense.xcresult TestApp.xcresult
```

Comment:
We might also want to include `TestAppiPadOS.xcresult` in here to make it complete.